packages/eslint-config-angular/index.js

```diff
@@ -40,6 +40,7 @@ module.exports = {
     '@angular-eslint/prefer-output-readonly': 'error',
     '@angular-eslint/relative-url-prefix': 'error',
     '@angular-eslint/use-injectable-provided-in': 'error',
+    '@angular-eslint/prefer-standalone-component': 'warn',
```

Comment:
What's the reason from going from error to warn? Or was there no error at all in the first place?

Reply:
Default config does not include this rule. Only a warning since there are valid cases for non-standalone components.